ops/daniel-kit/2_run_proof.ps1

```
@@ -0,0 +1,197 @@
+<#
+.SYNOPSIS
+    Step 2: Run contract proof and collect proof bundle artifacts.
+.DESCRIPTION
+    1. Runs 18-check contract test (writes contract-checks.log)
+    2. Starts the .NET API (if not already running)
+    3. Hits /ops/pacs/proof (writes proof.json)
+    4. Captures environment state (writes environment.txt)
+    All artifacts written to proof-bundle/ directory.
+#>
+param(
+    [string]$SaPassword = "TF_Pacs2026!",
```

PSScriptAnalyzer (1.24.0)

[warning] Parameter '$SaPassword' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $SaPassword type to SecureString; Set $SaPassword type to PSCredential

(PSAvoidUsingPlainTextForPassword)